ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/deprecated/mapper/response/PaymentIntentionMapperOld.java

```diff
@@ -10,12 +10,12 @@
 import uk.gov.hmcts.cmc.domain.models.response.PaymentIntention;
 
 @Component
-public class PaymentIntentionMapper implements Mapper<CCDPaymentIntention, PaymentIntention> {
+public class PaymentIntentionMapperOld implements Mapper<CCDPaymentIntention, PaymentIntention> {
```

Comment:
Seriously?

Reply:
Yes @srineelesh we agreed to remove instead as our approach.

Reply:
@ckalista: honestly wasn't intentional to keep it.. why would I keep this and delete all other deprecated classes?. I have deleted this now.